sdk/documentintelligence/azure-ai-documentintelligence/setup.py

```diff
@@ -13,7 +13,7 @@
 
 
 PACKAGE_NAME = "azure-ai-documentintelligence"
-PACKAGE_PPRINT_NAME = "Azure AI Document Intelligence"
+PACKAGE_PPRINT_NAME = "Azure Ai Documentintelligence"
```

Comment:
It seems like we should revert this change

Reply:
Reverted.